Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -48,6 +48,8 @@ class TabManagerImplementation: LegacyTabManager, Notifiable {
               forced || tabs.isEmpty
         else { return }
 
+        logger.log("Tabs restore started with \(forced), \(tabs.isEmpty)", level: .debug, category: .tabs)
```

Comment:
nit: without a string describing the variables here it's going to be hard to read without having to reference the code.